src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="/"
+    method="get"
+    data-qa="big"
+    >
+      <label for="search__form">
+        <input
+          class="search__field"
+          type="search"
+          name="searchBig"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
```

Comment:
we can use clearer name since label is not a form
```suggestion
      <label for="search__label">
        <input
          class="search__field"
          type="search"
          name="searchBig"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
      </label>
    </form>
```